capa/features/extractors/helpers.py

```diff
@@ -65,6 +66,34 @@ def generate_api_features(apiname, va):
             yield API(impname[:-1]), va
 
 
+def is_ordinal(symbol):
+    return symbol[0] == "#"
+
+
+def generate_import_features(dll, symbol, va):
+    """
+    for a given dll, symbol, and address, generate import features.
+    we over-generate features to make matching easier.
+    these include:
+      - kernel32.CreateFileA
+      - kernel32.CreateFile
+      - CreateFileA
+      - CreateFile
+    """
+    # (kernel32.CreateFileA, 0x401000)
+    yield Import(dll + "." + symbol), va
+    # (CreateFileA, 0x401000)
+    if not is_ordinal(symbol):
```

Comment:
Isn't something like this also needed in the api function? I think `generate_api_features()` is currently it is returning things like `ord3` for visisect.